packages/backend/src/controllers/events.controller.ts

```
@@ -1,67 +1,99 @@
- * Crée un nouvel event
+ * Create a new event
  * @route POST /api/events
  * @access Admin only
+ * @param req Express Request
+ * @param res Express Response
+ * @returns 201 + Event | 400 | 500
  */
-export const createEvent = async (req: Request, res: Response): Promise<void> => {
-  const event = await eventsService.createEvent(req.body);
-  res.status(201).json(event);
+export const createEvent = async (req: Request, res: Response): Promise<Response | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)